src/locale/nn.yml

```diff
@@ -713,7 +713,7 @@ feat:
           preview:
             callassignment_goal: fullføringsgruppa for ringeoppdraget "{queryTitle}"
             callassignment_target: målgruppa for ringeoppdraget  "{queryTitle}"
-            standalone: smartsøket
+            standalone: smartsøket "{queryTitle}"
```

Comment:
Thinking about if there could be some way that the translator interface/Lyra could indicate whether a translation should begin with big or small letter, if that could be helpful.